provisioning/playbooks/elastic/setup_kibana.yml

```diff
@@ -0,0 +1,41 @@
+---
+- hosts: elastic
+  sudo: True
+  vars:
+    kibana_version: 4.0.1
```

Comment:
Same reason as the couchbase create_bucket file, these variables are specific to setting up Kibana.